azavea/climate-change-lab
# Make sidebar sections collapsible

## Overview

Accordion sidebar temperature and precipitation section headers.
Give same hover styling as sub-elements (blue, with pointer cursor).

Open by default.


### Demo

After clicking on first header to collapse:
![image](https://user-images.githubusercontent.com/960264/32397019-23ce85ca-c0be-11e7-9204-1a8fee37aa78.png)


### Notes

Also minor version bump to `ngx-bootstrap` library, not necessitated by anything.

## Testing Instructions

 * `yarn install && yarn run serve`
 * Sidebar headers should change to blue with pointer cursor on hover to indicate clickable
 * Clicking sidebar header should expand/collapse section
 * Sections should both be expanded by default

## Checklist


Closes #300.
